yivi-docs/blog/2025-03-07-keyshare-migration/index.md

```diff
@@ -0,0 +1,58 @@
+---
+slug: 2025-arf-1-5
+title: 📢 Important notice Keyshare server migration
+authors: [dibranmulder]
+tags: [yivi, migration]
+---
+
+In the night of March 17th to 18th, 2025, at 01:00 CET, we will complete the final phase of the migration from SIDN to Caesar Groep. The last two components that require temporary downtime are the Keyshare Server and MyYivi. These systems play a crucial role in Yivi's authentication and identity management services, and we want to ensure a smooth transition with minimal disruption.
+
+## Migration Plan
+To ensure a seamless transition, we have prepared the following steps:
+
+- Yivi App Update - March 11th, 2025
+  - A Yivi App release will update the Keyshare Server endpoint from `https://irma.sidn.nl/tomcat/irma_keyshare_server` to `https://keyshare.yivi.app`.
+  - The trust scheme will be changed to use the new Keyshare Server.
+  - During this transition, https://keyshare.yivi.app will act as a reverse proxy to the existing SIDN Keyshare Server, ensuring uninterrupted service.
+- Final Migration - March 18th, 2025
+  - At 01:00 CET, the SIDN Keyshare Server will be shut down.
+  - The Keyshare database will be migrated to our new infrastructure.
+  - The reverse proxy at `https://keyshare.yivi.app` will be removed, and the new Keyshare Server will be brought online.
```

Comment:
Shouldn't the order of these to steps be in reverse?

Reply:
No, the downtime seems to be intended to reduce the complexity of the migration. First the old server is shutdown, the database is transferred to the new server, and the new server is brought online.